faulthandler.pth

```diff
@@ -0,0 +1 @@
+import os, sys; exec("if os.environ.get('PYTHONFAULTHANDLER'):\n import faulthandler\n faulthandler.enable()")
```

Comment:
Because this is not valid syntax:
```
import foo; if blabla: do(stuff)
```
The `if` is disallowed in multistatement lines.